### DISCUSSION TITLE
Just the first line is being coverted to html

### DISCUSSION THREAD

**Discussion by brunobandev** [spatie/laravel-markdown#19] [General]
Hi lads,

not sure if I'm using the library in the right way, but the fact is when I try to render markdown inside my blade page, just the first line is parsed, the rest keep being markdown tags. I followed the same process from here https://spatie.be/docs/laravel-markdown/v1/using-the-blade-component/general-usage.

My result is something like this: https://www.awesomescreenshot.com/image/14009144?key=534c5ff617f64b440a2178c46a81cf04

Thank you very much

**Comment by Plytas** [comment]
You're probably missing a line break. If you're using HEREDOC then leave an empty line before closing it.

**Comment by JSn1nj4** [comment]
I came looking to see if this had been talked about yet. Same issue for me on the latest version.

Only way around it is to start the markdown content all the way at the left of open editor.

I think what's happening is the indentation at the beginning of each line _after_ the first is being treated as part of the content, preventing the lines from being parsed as expected.

**Reply by JSn1nj4** [reply]
Actually, I stand corrected. It seems the indentation is causing it to interpret everything after the first line as a code block (thanks @rbarden).

It just didn't render like that until I installed shiki.

![image](https://user-images.githubusercontent.com/5084820/190882077-a57c3e06-401e-4f38-8a8d-b577608c348c.png)
